packages/datatrak-web-server/src/routes/SurveysRoute.ts

```diff
@@ -7,13 +7,13 @@ import { Request } from 'express';
 import camelcaseKeys from 'camelcase-keys';
 import sortBy from 'lodash.sortby';
 import { Route } from '@tupaia/server-boilerplate';
-import { DatatrakWebSurveyRequest } from '@tupaia/types';
+import { DatatrakWebSurveyRequest, Survey } from '@tupaia/types';
 
-type Survey = DatatrakWebSurveyRequest.ResBody;
+type SingleSurveyResponse = DatatrakWebSurveyRequest.ResBody;
 
 export type SurveysRequest = Request<
   DatatrakWebSurveyRequest.Params,
-  Survey[],
+  SingleSurveyResponse[],
```

Comment:
Out of curiosity, why do we make this into an array here instead of `ResBody` already being defined as an array in `types`?

Reply:
That is a good question. I was sure so I looked into it. We are using the same request types for SingleSurveyRoute and SurveysRoute which I think happened when we moved the single survey route to use the same central-server endpoint as the multiple one. It's good in a way that we are re-using types but could do with a clean up. I've added it to my upnext list rather than doing it now as this fix is blocking testing.